docs/content.md

```diff
@@ -4,83 +4,79 @@
 
 ###  Global
 
-* Design Principles
-* Accessibility
-* Voice & tone (off link to Manual of Style?)
-* Designing for Safety
+* Accessibility: consider a11y in design early, list of tools
+* Design principles
+* Designing for safety
+* Content
 
-### Foundation
+### Foundations
```

Comment:
Would we want to include an overview for this section to connect with https://github.com/primer/primitives?

Reply:
That's a good idea.